### PR TITLE
chore(server): #94 Added memcached service

### PR DESCRIPTION
**Pull request by deeptibaghel**
Added memcached.

### CODE REVIEW

**Comment by deeptibaghel** [comment]
@vladikoff , have added memcached, tested on mac only as yet. Pl. let me know if it looks ok.

File: _scripts/memcached.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const spawn   = require('child_process').spawn;
```
Comment (vladikoff) [inline _scripts/memcached.js]: multiple spaces after `spawn`, should just be one

File: _scripts/memcached.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+const memcachedProcess = spawn('memcached', ['-vv']);
```
Comment (vladikoff) [inline _scripts/memcached.js]: you need to pass an option here to inherit output from the spawn, `{ stdio: 'inherit' }` , see https://nodejs.org/api/child_process.html

Comment (vladikoff) [reply _scripts/memcached.js]: #### You goal here is to get the same output when running `./pm2 logs 16` and `memcached -vv` manually in the terminal:

image

**Comment by vladikoff** [comment]
@deeptibaghel almost there! just need a few fixes, also could you please rename the commit to `Adding memcached service`. This is not adding any docker services yet. We can merge this with just memcached for now 👍

**Comment by vladikoff** [comment]
@deeptibaghel there is also a test file here: https://github.com/mozilla/fxa-local-dev/blob/master/test/curl.sh

Add a new check to make sure memcache is running in there, I suggest running it locally first so you don't wait on GitHub/Travis CI to report results, it will take some time. Let me know if you have questions!

**Comment by deeptibaghel** [comment]
Thanks @vladikoff , made changes and tested :)

**Comment by vladikoff** [comment]
👍 Thank you!!

**Pull request merged**